src/syscall.h

```diff
@@ -8,14 +8,36 @@ See the file LICENSE for details.
 #define SYSCALL_H
 
 #include "kerneltypes.h"
+#include "graphics.h"
```

Comment:
I'm on board with Kyle here.
One crazy large, perhaps disorganized or difficult to keep organized file of random structs doesn't sound too appealing. Or at the very least instead of global_structs we should call it sys_structs.h or something to give an idea of what belongs in the file.